src/System.Private.CoreLib/src/System/Threading/ThreadPool.cs

```diff
@@ -37,6 +37,20 @@ internal static class ThreadPoolGlobals
         public static bool enableWorkerTracking;
 
         public static readonly ThreadPoolWorkQueue workQueue = new ThreadPoolWorkQueue();
+
+#if CORECLR
+        /// <summary>Shim used to invoke <see cref="ITaskCompletionAction.Invoke"/> of the supplied <see cref="IAsyncStateMachineBox"/>.</summary>
```

Comment:
Nit: this comment was wrong before and is still wrong in the new location :) Copy and paste error on my part.  It should be "IAsyncStateMachineBox.MoveNext", not "ITaskCompletionAction.Invoke".